plugins/outputs/plugin.go

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2022 Aspect Build Systems, Inc.
+ *
+ * Licensed under the aspect.build Commercial License (the "License");
+ * you may not use this file except in compliance with the License.
+ * Full License text is in the LICENSE file included in the root of this repository.
+ */
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	goplugin "github.com/hashicorp/go-plugin"
+
+	"aspect.build/cli/pkg/bazel"
+	"aspect.build/cli/pkg/interceptors"
+	"aspect.build/cli/pkg/ioutils"
+	"aspect.build/cli/pkg/plugin/sdk/v1alpha2/config"
+	aspectplugin "aspect.build/cli/pkg/plugin/sdk/v1alpha2/plugin"
+)
+
+func main() {
+	goplugin.Serve(config.NewConfigFor(NewDefaultPlugin()))
+}
+
+type Outputs struct {
+	aspectplugin.Base
+	ioutils.Streams
+	Bzl bazel.Bazel
```

Comment:
I dont think this is actually being used? A `bzl` instance comes through with the function callback now

Reply:
yeah good call, the streams wasn't used anymore either